tests/test_db_store.py

```diff
@@ -3,6 +3,7 @@
 
 import mock
 import datetime
```

Comment:
`datetime` should be in the first group of imports because it's a standard library module
https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting